lib/hpcbase.pm

```diff
@@ -272,7 +272,7 @@ sub check_nodes_availability {
     my ($self) = @_;
     my @cluster_nodes = cluster_names();
     foreach (@cluster_nodes) {
-        assert_script_run("ping -c 3 $_");
+        script_sudo("ping -c 3 $_");
```

Comment:
What was the failure here, that you moved to `script_sudo` instead of `assert_script_run`? you could wait longer between pings, by using `ping -i $interval_in_seconds -c $count $_`, mind the default timeout for assert_script_run.

Looking at [this](http://aquarius.suse.cz/tests/13115#step/mpi_master/64), looks like you're relying on a behavior to avoid the test failing (not intentionally) and not even checking the output from script_sudo... maybe for what you want to do, `script_retry` is a better choice.

Reply:
Let me ping you tomorrow about that. i will need to take another look myself too. I did use `-i` in the beginning of this PR but i remove it at the end. But i am not sure i am thinking clear right now. But i feel your input is on the right direction. Thank you

Reply:
i created https://github.com/os-autoinst/os-autoinst/pull/2191 for the failure of `assert_script_sudo`.